testomania/src/main/java/com/earth/testomania/presentation/AboutBottomSheet.kt

```diff
@@ -72,6 +82,25 @@ fun AboutBottomSheet() {
     developers.forEach {
         AboutDeveloper(it)
     }
+
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(bottom = 10.dp),
+        contentAlignment = Alignment.Center
+    ) {
+        Icon(
```

Comment:
Why not just the Icon with .fillMaxWisth?

Reply:
Ok let me try

Reply:
yep works with fillMaxWidth, but it click now is clickable whole width and click animation is also stretched
![image](https://user-images.githubusercontent.com/6556678/182040568-fe6b621c-3c0e-4250-bfa9-14dcd6350bac.png)

Reply:
Disable ripple animation, now it works